components/brave_rewards/browser/service_sandbox_type.h

```diff
@@ -17,11 +17,11 @@ class BatLedgerService;
 template <>
 inline sandbox::mojom::Sandbox
 content::GetServiceSandboxType<bat_ledger::mojom::BatLedgerService>() {
-#if !defined(OS_ANDROID)
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
I think we need the include in this file?

Reply:
Somehow I missed this one too :facepalm:. Added in https://github.com/brave/brave-core/commit/ffb4a13308e6ebe0c8af573edab9f32f69406a71